trestle/oscal/common.py

```diff
@@ -1304,13 +1302,13 @@ class Config:
         description='A name given to the document revision, which may be used by a tool for display and navigation.',
         title='Document Title',
     )
-    published: Optional[Published] = None
-    last_modified: Optional[LastModified] = Field(None, alias='last-modified')
-    version: Version
+    published: Optional[datetime] = None
+    last_modified: Optional[datetime] = Field(None, alias='last-modified')
+    version: constr(regex=r'^\S(.*\S)?$')
```

Comment:
class Version can then be deleted?

Reply:
I'd like to leave all original classes in place to show how it was originally laid out.  I will make an issue to possibly remove them later.  The classes as shown are the original and intended ones - and capture the original structure, which I think is helpful to convey the context of the regex.